src/hooks/useLoadAccountData.ts

```diff
@@ -23,17 +22,16 @@ export default function useLoadAccountData() {
       // tokens + nfts
       if (network === networkTypes.mainnet) {
         const p1 = dispatch(dataLoadState());
-        const p2 = dispatch(uniqueTokensLoadState());
```

Comment:
this redux action got nfts from local storage and put them into redux. this happens automatically in react query